Cargo.toml

```diff
@@ -1,30 +1,30 @@
 [package]
-name = "pallet-perun"
-version = "3.0.0"
 authors = ["PolyCrypt GmbH <info@polycry.pt>"]
+description = "FRAME pallet for Perun State Channels"
 edition = "2018"
-license = "Apache-2.0"
 homepage = "https://polycry.pt/"
 keywords = ["blockchain", "channel", "perun"]
-repository = "https://github.com/perun-network/perun-polkadot-pallet"
-description = "FRAME pallet for Perun State Channels"
+license = "Apache-2.0"
+name = "pallet-perun"
```

Comment:
Why did you reorder? I think the ordering before appeared more natural, name and version at top.

Reply:
The TOML formatter that i used ordered them alphabetically.

Reply:
OK, I don't know. I think in this case ordering by (subjective) relevance would be more suitable. Is that a special kind of formatter you are using or a common thing to do in rust projects?

Reply:
Not that i know of. Just a normal TOML formatter. I reverted the changes.